source/AuditLog/AuditLogClient/AuditLogHttpClient.cs

```diff
@@ -41,6 +41,8 @@ public async Task LogAsync(
         Guid logId,
         Guid userId,
         Guid actorId,
+        string? actorNumber,
+        string? marketRoles,
```

Comment:
```suggestion
        string? marketRole,
```

I do believe that "market role" is a proper english word?

Reply:
it is a semicolon separated string :)